lib/util/template/index.ts

```diff
@@ -8,6 +8,10 @@ import { regEx } from '../regex';
 handlebars.registerHelper('encodeURIComponent', encodeURIComponent);
 handlebars.registerHelper('decodeURIComponent', decodeURIComponent);
 
+handlebars.registerHelper('encodeBase64', (str: string) =>
+  Buffer.from(str).toString('base64'),
```

Comment:
```suggestion
  Buffer.from(str ?? '').toString('base64'),
```

should we protect against null or undefined?

Reply:
This has probably impacted code coverage now :)

Reply:
I thought about this, but then I didn't see other helpers do that, so I figured I'd follow the same pattern for now.